metrics/nop_reporter.go

```diff
@@ -99,3 +103,30 @@ type nopCachedTimer struct {
 
 func (nopCachedTimer) ReportTimer(interval time.Duration) {
 }
+
+// NopCachedHistogram is an implementation of tally.CachedHistogram
+var NopCachedHistogram tally.CachedHistogram = nopCachedHistogram{}
+
+type nopCachedHistogram struct {
+}
+
+func (nopCachedHistogram) ValueBucket(
+	bucketLowerBound, bucketUpperBound float64,
```

Comment:
style nit: inconsistent oneline versus multiline. we may need a style guide :|

i thought we either did:

```go
func foo(lotsAndLots, ofArgumentsHere ofThisType) {
```

or
```go
func foo(
  lotsAndLots,
  ofArguments ofThisType,
)
```

but maybe i'm misremembering